packages/a/aws-sdk-cpp/xmake.lua

```diff
@@ -0,0 +1,68 @@
+package("aws-sdk-cpp")
+    set_homepage("https://github.com/aws/aws-sdk-cpp")
+    set_description("AWS SDK for C++")
+
+    add_urls("https://github.com/aws/aws-sdk-cpp.git")
+    add_versions("1.9.362", "e9372218a2c8fab756ecaa6e4fefcdb33c3670c1")
+
+    if is_plat("macosx") then
+        add_frameworks("Foundation", "CoreFoundation", "Security", "SystemConfiguration")
+    end
```

Comment:
Why add it repeatedly? Need it even if http is not enabled?

Reply:
> Why add it repeatedly? Need it even if http is not enabled?

deleted. 🧠🔫 😂